lib/synonyms.js

```diff
@@ -73,7 +73,5 @@ module.exports = multiword({
     'notLike', 'isNotLike', 'doesNotHave', 'isNotSimilar', 'isDissimilar'
   ],
 
-  type: [
-    'isa', 'isA'
-  ]
+  type: ['isA']
```

Comment:
Removed `isa` as the multiword function was generating it from `isA` so we ended up with duplicate of `isa`.